test/mpi/datatype/struct_pack_mpi_bottom.c

```diff
@@ -0,0 +1,224 @@
+/* -*- Mode: C; c-basic-offset:4 ; indent-tabs-mode:nil ; -*- */
+/*
+ *  (C) 2022 by Argonne National Laboratory.
+ *      See COPYRIGHT in top-level directory.
+ *
+ *  This code was written by ParTec AG.
+ *  Copyright (C) 2022      ParTec AG.  ParTec provides this material
+ *  Argonne National Laboratory subject to MPICH Corporate Contributor
+ *  License Agreement dated September 30, 2019.
+ */
```

Comment:
We have simplified the copyright block. Please refer to examples in other source files. Individual companies are acknowledged in the `COPYRIGHT` file. Please add a commit adding "ParTec AG" to the list of EXTERNAL CONTRIBUTIONS.

Reply:
Fixed this.

Reply:
@hzhou is there still any blocker to this PR? Did you already have the chance to run the CI tests with GPU enabled?